recipes/diffpy.snmf/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{%- set name = "diffpy.snmf" -%}
+{%- set version = "0.1.0" -%}
+{%- set sha256 = "2903bd7d9df588b35bd45bc07ea745c69fb281fb774beef8af645cebcb980b7c" -%}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/d/diffpy_snmf/diffpy_snmf-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  preserve_egg_dir: true
```

Comment:
What's the reason for this?

Reply:
This was put there to prevent our diffpy. namespace getting clobbered.  We are looking into whether it is needed or not but this will need some testing.

Reply:
@xhochy after extensive testing we are confident we can remove this, so it is gone in the current version.